bin/git-submitpr

```diff
@@ -115,13 +125,21 @@ cleanup_remote_branch() {
   exit 1
 }
 
+gitlab_create_pr() {
+  native_open "$gitlab_url/new?merge_request[source_branch]=$1&merge_request[target_branch]=$2"
+}
+
 error_file=$(mktemp)
 if git -C "$worktree_dir" remote get-url mine 2>/dev/null && git -C "$worktree_dir" push --no-verify $quiet_arg --set-upstream mine "$branch_name"; then
```

Comment:
does this pattern in general make sense with gitlab? this is here to support forks where you have `origin` as the upstream and a remote named `mine` as your fork. i haven't used gitlab in a while, does it require forks in the way github does for open source repo contributions?

Reply:
As I understand, GitLab [works](https://docs.gitlab.com/ee/user/project/repository/forking_workflow.html) in the same way:
>If you do not have write access for the repository you want to contribute to, you can create a fork.
>Make changes in your own fork and submit them through a merge request to the repository you don’t have access to.

So it makes sense to do it for GitLab too.